src/models/User.js

```diff
@@ -11,7 +11,7 @@ const UserSchema = new Schema(
     password: { type: String, required: true },
     avatar: {
       type: String,
-      required: false,
+      required: true,
       default: 'http://localhost:5000/statics/avatar/dog.png',
```

Comment:
Do wee need this required? It is being created anyways because we have a default. I just remove the required field.

Reply:
then we dont need it in the base url as well?

Reply:
so we could delete this branch ?

Reply:
Yes, we can close it.